conf/config-default.yaml

```diff
@@ -28,6 +28,7 @@ apisix:
   #   - ip: 127.0.0.2               # Specific IP, If not set, the default value is `0.0.0.0`.
   #     port: 9082
   #     enable_http2: true
+  # listen_backlog: 511             # Sets nginx listen backlog.If not set, the default value is `511`. http://nginx.org/en/docs/http/ngx_http_core_module.html#listen
```

Comment:
Like `enable_http2`, it would be better to make the `listen_backlog` as a per-listen configuration.